nvim/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Vim config
+mkdir -p ~/.config/nvim
+ln -sf $(pwd)/.vimrc ~/.config/nvim/init.vim
```

Comment:
pwd return different values depending on path from which the script was run, not the script home

Reply:
Fixed now